.eslintrc.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  env: {
+    browser: true,
+    commonjs: true,
+    es6: true,
+    node: true,
+  },
+  extends: [
+    'plugin:react/recommended',
+    'airbnb',
+  ],
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly',
+  },
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: 2018,
+  },
+  plugins: [
+    'react',
+  ],
+  rules: {
+  },
```

Comment:
Minor suggestion - perhaps you can get rid of this entirely if you have no rules to specify.

Reply:
Good point! I'll do that tomorrow.